src/main/java/cinema/config/SecurityConfig.java

```diff
@@ -1,41 +1,47 @@
 package cinema.config;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.annotation.Bean;
+import org.springframework.http.HttpMethod;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
-import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.core.userdetails.UserDetailsService;
 
 @EnableWebSecurity
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+    private final UserDetailsService userDetailsService;
+
+    public SecurityConfig(UserDetailsService userDetailsService) {
+        this.userDetailsService = userDetailsService;
+    }
+
     @Autowired
     public void configureGlobal(AuthenticationManagerBuilder authBuilder) throws Exception {
         authBuilder
-                .inMemoryAuthentication()
-                .passwordEncoder(getEncoder())
-                .withUser("user")
-                .password(getEncoder().encode("1111"))
-                .roles("USER");
+                .userDetailsService(userDetailsService);
```

Comment:
тут також варто додати PasswordEncoder
```suggestion
                .userDetailsService(userDetailsService)
                .passwordEncoder(passwordEncoder);
```